server/src/weaverbird/backends/pypika_translator/translators/postgresql.py

```diff
@@ -45,5 +47,24 @@ def duration(
         )
         return query, StepTable(columns=[*table.columns, step.new_column_name])
 
+    def customsql(
+        self: Self, *, step: "CustomSqlStep", table: StepTable
+    ) -> tuple["QueryBuilder", StepTable]:
+        """create a custom sql step based on the current table named ##PREVIOUS_STEP## in the query"""
+
+        if table.name is None:
+            raise MissingTableNameError()
+
+        # We want all columns wraps in semicolumns
+        table.columns = [f'"{c}"' for c in table.columns]
```

Comment:
This should be up to PyPika

Reply:
True, i handled that here : https://github.com/ToucanToco/weaverbird/pull/1390/commits/570db8145897639ba5e0328258cb13cf6ccd969f